src/geom/Location.js

```diff
@@ -953,6 +953,40 @@ define([
                 return lat;
         };
 
+        /**
+         * Computes a new set of locations translated from a specified location to a new location.
+         *
+         * @param {Globe} globe Globe used to compute translation.
+         * @param {Location} oldLocation The original reference location.
+         * @param {Location} newLocation The new reference location.
+         * @param {Location[]} locations The locations to translate.
+         *
+         * @return {Location[]} The translated locations.
+         */
+        Location.computeShiftedLocations = function(globe, oldLocation, newLocation, locations)
```

Comment:
This is ported from Java as is, but wouldn't it better located in globe as a non-static method?

Reply:
@claudiaifrim, we've discussed this with @pdavidc today. The logic here is more of the responsibility of globe, which knows about the projection, than location, which is simply a class representing a location (holding data). Globe already has this type of methods for other actions. We also see benefits in removing the double dependency this creates between Globe and Location. Consequently, could you move it as a non-static method of Globe?